### PR TITLE
THE PENCIL HOLDER

### PR DESCRIPTION
**Pull request by wonderinghost**
this is a WIP. im not a experienced coder so i have no idea what to do to fix shit.

# Document the changes in your pull request

# Spriting

# Wiki Documentation

# Changelog

:cl:  
rscadd: Added new things  
rscdel: Removed old things  
bugfix: fixed a few things  
wip: This is work in progress  
tweak: tweaked a few things  
soundadd: added a new sound thingy  
sounddel: removed an old sound thingy  
imageadd: added some icons and images  
imagedel: deleted some icons and images
mapping: This is a mapping change
spellcheck: fixed a few typos  
experimental: This is experimental
/:cl:

### CODE REVIEW

**Comment by wonderinghost** [comment]
have no idea how to code. also didnt know drafts still come up on discord so

File: code/modules/paperwork/pencil_holder.dm [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		/obj/item/pen/fountain/captain,
+		/obj/item/toy/crayon/red,
+		/obj/item/toy/crayon/orange,
+		/obj/item/toy/crayon/yellow,
+		/obj/item/toy/crayon/green,
+		/obj/item/toy/crayon/blue,
+		/obj/item/toy/crayon/purple,
+		/obj/item/toy/crayon/black,
+		/obj/item/toy/crayon/rainbow,
+		/obj/item/toy/crayon/mime,
```
Comment (Moltijoe) [inline code/modules/paperwork/pencil_holder.dm]: don't need to include every single type
/obj/item/pen will include all subtypes of it, that extends to /obj/item/toy/crayon

File: code/modules/paperwork/pencil_holder.dm [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+	new	/obj/item/pen (src)
+	new	/obj/item/pen/red (src)
+	new	/obj/item/pen/red (src)
+	new	/obj/item/pen/red (src)
+	new	/obj/item/pen/blue (src)
+	new	/obj/item/pen/blue (src)
+	new	/obj/item/pen/blue (src)
+	new	/obj/item/pen/green (src)
+	new	/obj/item/pen/green (src)
+	new	/obj/item/pen/green (src)
```
Comment (Moltijoe) [inline code/modules/paperwork/pencil_holder.dm:44]: why not make /obj/item/pen/random that picks a random coloured pen and spawn it instead?

**Review by Moltijoe — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/pencil_holder.dm (hunk 2 above)
Comment (wonderinghost) [reply code/modules/paperwork/pencil_holder.dm:44]: I have no idea if at some point it tries to spawn in the traitor items. Also more so uniformity.

I also updated the maint loot table in a prior pr and I have no idea if different colored pens are more common in maint now. So it is a guarantee that they will spawn in if they dont

**Comment by wonderinghost** [comment]
have a different pr that im trying to figure out how to sterilize

**Pull request closed without merging**